SU-SWS/jumpstart_ui
# D8CORE-2444: Changing the anchor to a div for the landing of the anchor

# READY FOR REVIEW
- I'd like to get some commentary on how to make this better or a different approach.  - DONE
- Removed all the JS changes and move to PHP.
- Received feedback from Clare from SOAP.
- Needs a review by @cjwest 

# Summary
- Changing the skip link target from an anchor to a div

# Needed By (Date)
- 4/12

# Urgency
- Med

# Steps to Test

1. Checkout this branch and pull in this change.
2. Clear the caches.
3. Got to a page on your site in an incognito mode. 
4. Press Tab first to get the Skip to main link.
5. Press Enter when when the Skip Link has focus.
6. Notice you jump to a div.
7. If you press "Option"+"Control"+"Shift"+U,  notice that you can get to the Main area in the Landmarks Menu. - only when. you have VoiceOver on.

# Affected Projects or Products
- JumpstartUI
- There should be no change visible on the UI unless you are using `tab` key to navigate.

# Associated Issues and/or People
- D8CORE-2444
- A11y
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
